Documentation/config/core.txt

```diff
@@ -626,3 +626,6 @@ core.abbrev::
 	in your repository, which hopefully is enough for
 	abbreviated object names to stay unique for some time.
 	The minimum length is 4.
+
+core.parallelCheckout::
```

Comment:
I wonder if this should go somewhere other than `core.*`.

Perhaps `checkout.parallel`? or `parallel.checkout`?

Reply:
probably, yes.